models/QuickBuilder.cfc

```diff
@@ -673,7 +673,9 @@ component extends="qb.models.Query.QueryBuilder" accessors="true" {
 			return arguments.column;
 		}
 
-		return listLast( getFrom(), " " ) & "." & getEntity().retrieveColumnForAlias( arguments.column );
+		return getEntity().isParentAttribute( arguments.column )
+				? getEntity().retrieveColumnForAlias( arguments.column )
```

Comment:
I had that in there before and can't remember why I took it out now. It might not be a bad idea if the child class has a column that overloads the parent.  I'm not sure how qB data will normalize that column name in the query, though.  I'll make the change after a few tests.

Reply:
Done